tests/Unit/Utilities/Test_GetOutput.cpp

```diff
@@ -0,0 +1,13 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "Utilities/GetOutput.hpp"
+
+SPECTRE_TEST_CASE("Unit.Utilities.GetOutput", "[Utilities][Unit]") {
+  CHECK(get_output(3) == "3");
+  CHECK(get_output(std::string("abc")) == "abc");
```

Comment:
I think it would be good to test something that is not by default streamable.

Reply:
That's done in some other PR.  I'll find the relevant stuff and cherry-pick it here.